package.json

```
@@ -46,9 +46,9 @@
   "main": "preset.js",
   "name": "storybook-addon-next-router",
   "peerDependencies": {
-    "@storybook/addon-actions": "^6.0.0",
-    "@storybook/addons": "^6.0.0",
-    "@storybook/client-api": "^6.0.0",
+    "@storybook/addon-actions": "^6 || >=6.0.0-0 || >=6.1.0-0 || >=6.2.0-0 || >=6.3.0-0 || >=6.4.0-0",
+    "@storybook/addons": "^6 || >=6.0.0-0 || >=6.1.0-0 || >=6.2.0-0 || >=6.3.0-0 || >=6.4.0-0",
+    "@storybook/client-api": "^6 || >=6.0.0-0 || >=6.1.0-0 || >=6.2.0-0 || >=6.3.0-0 || >=6.4.0-0",
```

```suggestion
    "@storybook/addon-actions": ">=6.0.0",
    "@storybook/addons": ">=6.0.0",
    "@storybook/client-api": ">=6.0.0",
```

@benbender does the suggestion work for you as well?

---

@lifeiscontent That's the failing config :) Context is automatic installation and strict enforcing of peer-deps in npm 7. See https://github.blog/2021-02-02-npm-7-is-now-generally-available/#peer-dependencies

---

@benbender how are you able to even get storybook running? Last time I checked the peer deps were really messed up. do you mind putting a small project together so I can test this change?